composer.json

```diff
@@ -11,6 +11,7 @@
 		"php": ">=7.0"
 	},
 	"require-dev": {
+		"ext-mbstring": "*",
```

Comment:
Is mbstring actually used outside of dev/tests environment? If yes, that's a problem for inclusion in WordPress core and we'll need to use the PHP UTF-8 decoder. If not, we're good.

Reply:
@adamziel Only in tests. I'm using it when testing the MySQL identifier Unicode range.